backend/src/main/java/it/compare/backend/product/response/ProductListResponse.java

```diff
@@ -0,0 +1,17 @@
+package it.compare.backend.product.response;
+
+import it.compare.backend.product.model.Category;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class ProductListResponse {
+    private String id;
+    private String name;
+    private Category category;
+    private String mainImageUrl;
+    private BigDecimal lowestCurrentPrice;
+    private String lowestPriceShopName;
```

Comment:
Remove `Name` suffix to be consistent.
```suggestion
    private String lowestPriceShop;
```

Reply:
done